python/ray/tests/test_resource_demand_scheduler.py

```diff
@@ -3555,10 +3559,11 @@ def test_info_string_with_launch_failures():
  2.00GiB/8.00GiB memory
  3.14GiB/16.00GiB object_store_memory
 
+Constraints:
+ {'CPU': 16}: 100 from request_resources()
 Demands:
  {'CPU': 1}: 150+ pending tasks/actors
```

Comment:
Could you have a follow-up to remove the `+` for demands as well if the count is accurate.